profile/README.md

```diff
@@ -1,14 +1,28 @@
 <h1 align="center">EddieHub</h1>
 
-<p align="center">
-Most inclusive Open Source community on the planet!
-</p>
-
 <h2 align="center">Collaboration 1st, Code 2nd! </h2>
 <p align="center">
-<img src="https://github.com/EddieHubCommunity.png" align="center" height="200" />
+    <img src="https://github.com/EddieHubCommunity.png" align="center" height="200" />
 </p>
 
+Here, right now on **your** screens, you are in the presence of the most inclusive *Open Source* community on the planet!
+
+## Meet a few members...
+
+| Member | Photo | Role |
+| ------ | ---- | ----- |
+| Hubber | <img src="./images/hubber.png" height=90> | The Mascot! |
+| Chibi  | <img src="./images/chibi.png" height=90> | Discord bot! |
+| Eddie  | <img src="./images/eddie.jpg" height=90> | Founder of EddieHub! |
```

Comment:
I would suggest using the copy and past method or linking to the branding repo with the original images. Hope you understand what I mean.

For Eddie you could use `https://github.com/eddiejaoude.png` this will automatically get the current profile picutre of eddie

Reply:
Some more thougts about the "meet some of the members" section.

I'm also not sure if we need something like this in the Org readme. The respository like "Discord-Bot" is pinned.

I like the idea of introducing hubber but I think Hubber should not be listed as Member of our community.

Reply:
So should I remove that?

Reply:
Current readme looks like this:
![image](https://user-images.githubusercontent.com/30869493/134127578-ab68d2aa-2ebe-4e63-a707-e554e2b117f1.png)

Maybe only add one sentence above Hubber that "this is Hubber our maskot..."